packages/iov-keycontrol/src/wallets/ed25519wallet.spec.ts

```diff
@@ -36,6 +36,25 @@ describe("Ed25519Wallet", () => {
     expect(wallet.label.value).toBeUndefined();
   });
 
+  describe("previewIdentity", () => {
+    it("works", async () => {
+      const wallet = new Ed25519Wallet();
+      const newIdentity = await wallet.previewIdentity(defaultChain, defaultKeypair);
+      expect(newIdentity).toBeTruthy();
+      expect(newIdentity.pubkey.algo).toEqual(Algorithm.Ed25519);
```

Comment:
This is covered by the next assertion.

Reply:
Thanks, done 89b5e56d776946fb599ea9d999c943b824d6469e